modules/test/src/main/resources/logback.xml

```diff
@@ -14,7 +14,6 @@
   </root>
 
   <!-- Minimize infrastructure debug logs -->
-  <logger name="org.dozer" level="INFO"/>
```

Comment:
We migrated to a new dozer version that has a new namespace (with github in it).
As dozer unfortunately logs like hell by default (what is entirely nuts) we have to ensure that dozer is at least on info level and not on debug (otherwise the logs explode).
We can address this in a separate PR but I missed this during the dozer migration and this needs to be fixed urgently as well (mainly in the `logback.xml` of the archetype in `server` module).

Reply:
I have seen this in real-life projects where builds/releases failed due to OutOfMemoryError cause dozer was logging gigabytes of properties that have been mapped.